sql/src/main/java/io/crate/executor/transport/TransportExecutor.java

```diff
@@ -352,6 +352,7 @@ public Task visitMultiPhasePlan(MultiPhasePlan multiPhasePlan, final Void contex
                         consumer.accept(null, e);
                     }
                 });
+                SubSelectSymbolReplacer replacer = new SubSelectSymbolReplacer(rootPlan, entry.getValue());
```

Comment:
Code clarity. The `SubSelectSymbolReplacer` is only used in the next line and has no dependencies on any symbols above.